node/src/service.rs

```diff
@@ -190,7 +196,11 @@ pub fn new_full(config: Configuration) -> Result<TaskManager, ServiceError> {
         &config.chain_spec,
     );
     let (grandpa_protocol_config, grandpa_notification_service) =
-        sc_consensus_grandpa::grandpa_peers_set_config(grandpa_protocol_name.clone());
+        sc_consensus_grandpa::grandpa_peers_set_config::<_, Network>(
+            grandpa_protocol_name.clone(),
+            NotificationMetrics::new(None),
```

Comment:
Shouldn't we attach our Prometheus metrics here ?

Reply:
Didn't know we had Prometheus metrics, in that case yes, I think they should go here. I'll dig a bit more.

Reply:
@drgora